chapters/01-preface.Rmd

```diff
@@ -0,0 +1,334 @@
+
+Jun
+: Completed an [Insight Data Science][insight] fellowship last year after
+doing a PhD in Geology and now works for a company that does forensic audits. He
+uses a variety of machine learning and visualization packages, and would now
+like to turn some of his own work into an open source project. This book will
+show him how such a project should be organized and how to encourage people to
+contribute to it.
+
+Sami
+: Became a competent programmer during a bachelor's degree in applied math and
+was then hired by the university's research computing center. The kinds of
+applications they are being asked to support have shifted from fluid dynamics to
+data analysis; this guide will teach them how to build and run re-usable data
+analysis pipelines so that they can pass those skills on to their users.
+
+### Prerequisites
+
+Learners are expected to be able to:
+
+- Write reusable functions to analyze tabular data using [R][r].
+- Have a basic understanding of [Git][git] and [GitHub][github], and have at least
+used them for basic version control tasks.
+- Create reproducible reports using [R Markdown][r-markdown].
+
+Learners will need:
+
+- A personal computer with Internet access.
+- To have [R][r], [Git][git], and [RStudio][rstudio] installed.
+- A [GitHub][github] account.
+
+### What's the big picture?
+
+This book and the tools, skills, and knowledge being taught come out of the
+growing awareness of and demand for [open science][open-science],
+[reproducible research][reproducibility], and 
+[software sustainability][sustainable-software]. So what are these?
+
+- [Open science][open-science] is a movement to make scientific data, methods,
+and results publicly and freely accessible by publishing them with 
+[open licenses][open-license]. These licenses allow others to re-use and modify
+these outputs without worrying about copyright infringement, often only requiring
+attribution to the original source.
+- [Reproducible research][reproducible-research] means ensuring that anyone with
+access to data and software can feasibly reproduce the same or similar results,
+both to check them and to build on them. It's important to note that science can
+be open but not reproducible and reproducible but not open.
+- [Sustainable software][sustainable-software] is when software is developed in
+a way that makes it easier for other people to extend and/or maintain it, rather
+than to replace it. Sustainability isn't just about the software itself, it also
+depends on the skills and culture of its users. For instance, if a software
+package is being maintained by a couple of post-docs who are being paid a
+fraction of what they could earn in industry and have no realistic hope of
+promotion because their field doesn't value tool building, it doesn't matter
+whether the package is well tested and easy to install: sooner or later
+(probably sooner) it will become [abandonware][abandonware].
+
```

Comment:
Add a summary statement here: "This book presents software development that emphasizes both reproducibility and sustainability, because it supports both individual researchers and the entire research community."